skywalking-sniffer/skywalking-sdk-plugin/mongodb-plugin/pom.xml

```diff
@@ -0,0 +1,52 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>skywalking-sdk-plugin</artifactId>
+		<groupId>com.a.eye</groupId>
+		<version>3.0.1-2017</version>
+	</parent>
+
+	<artifactId>skywalking-mongodb-plugin</artifactId>
```

Comment:
As you only declared:
>Supported Version: mongo-java-driver-3.X.X
I have tested based on version mongo-java-driver-3.4.2

Your `artifactId` should be `skywalking-mongodb-3.x-plugin`.

Reply:
My project based on mongo
```
<dependency>
            <groupId>org.mongodb</groupId>
            <artifactId>mongo-java-driver</artifactId>
            <version>3.8.2</version>
        </dependency>
```
skywalking agent with apm-mongodb-3.x-plugin-6.1.0.jar not works,as I can't see any mongo trace log.